python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -986,7 +988,9 @@ def test_torch_tensor_nccl_all_reduce_get_partial(ray_start_regular):
             for i, worker in enumerate(workers)
         ]
         collectives = collective.allreduce.bind(computes, ReduceOp.SUM)
-        dag = workers[0].recv.bind(collectives[0])
+        recv = workers[0].recv.bind(collectives[0])
+        tensor = workers[1].recv_tensor.bind(collectives[0])
```

Comment:
Is the change of adding `tensor` is testing both metadata and tensor, while previously only testing metadata?

Reply:
Yes. Stephanie commented that we should check both.